examples/raw.rs

```diff
@@ -11,72 +11,106 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-extern crate futures;
-extern crate tikv_client;
-
+use futures::future::Future;
 use std::path::PathBuf;
+use tikv_client::{raw::Client, Config, Key, KvPair, Result, Value};
 
-use futures::future::Future;
-use tikv_client::*;
+const KEY: &str = "TiKV";
+const VALUE: &str = "Rust";
+const CUSTOM_CF: &str = "custom_cf";
```

Comment:
oh, can TiKV support this CF?

Reply:
If I remember it correctly, TiKV doesn't support customized cf setup. Maybe it should be supported at some point later on?

Reply:
maybe yes, but here I think we should use "default"

Reply:
Fixed.